src/BlazorWasmSdk/Tasks/FileHasher.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Numerics;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.NET.Sdk.BlazorWebAssembly
+{
+    public static class FileHasher
+    {
+        public static string GetFileHash(string filePath)
+        {
+            using var hash = SHA256.Create();
+            var bytes = Encoding.UTF8.GetBytes(filePath);
+            var hashBytes = hash.ComputeHash(bytes);
+            return ToBase36(hashBytes);
+        }
+
+        private static string ToBase36(byte[] hash)
+        {
+            const string chars = "0123456789abcdefghijklmnopqrstuvwxyz";
+
+            var result = new char[10];
+            var dividend = BigInteger.Abs(new BigInteger(hash.AsSpan().Slice(0, 9).ToArray()));
+            for (var i = 0; i < 10; i++)
+            {
+                dividend = BigInteger.DivRem(dividend, 36, out var remainder);
+                result[i] = chars[(int)remainder];
+            }
+
+            return new string(result);
+        }
```

Comment:
Just in case you're interested - and feel free to ignore this - we have some similar code elsewhere in the product like this:

```cs
            byte[] hashBytes;
            using (var sha1 = SHA1.Create())
            using (var fileStream = File.OpenRead(path))
            {
                hashBytes = sha1.ComputeHash(fileStream);
            }

            var hash = Convert.ToBase64String(hashBytes);
```

I don't really mind, but you could just use the first 10 chars from the base64 representation of the SHA1/SHA256 hash if you wanted. It doesn't seem like it case-sensitivity matters either way in this case.

Reply:
this should use the same thing we use in ComputeCssScope, since we did all the math for collisions on that

Reply:
https://github.com/dotnet/sdk/blob/main/src/RazorSdk/Tasks/ComputeCssScope.cs#L57-L69